apps/consoledot-rhosak/src/routes/data-plane/UseTopicConsumerGroupGate.tsx

```diff
@@ -0,0 +1,32 @@
+import { useConsumerGroup } from "consoledot-api";
+import { useRouteMatch } from "react-router-dom";
+import {
+  DataPlaneTopicConsumerGroupRoutePath,
+  DataPlaneTopicConsumerRouteParams,
+} from "./routesConsts";
+import { useTopicGate } from "./useTopicGate";
+
+export function useTopicConsumerGroupGate() {
+  const { instance, topic } = useTopicGate();
+  const match = useRouteMatch<DataPlaneTopicConsumerRouteParams>(
+    DataPlaneTopicConsumerGroupRoutePath
+  );
+  if (!match) {
+    throw Error("useTopicConsumerGroupGate used outside the expected route");
+  }
+  const { data: consumerGroup } = useConsumerGroup(
+    {
+      id: instance.id,
+      adminUrl: instance.adminUrl,
+      consumerGroupId: match.params.consumerGroupId,
+    },
+    true
+  );
+
+  return {
+    instance,
+    consumerGroup: consumerGroup as NonNullable<typeof consumerGroup>,
+    match,
+    topic,
+  };
+}
```

Comment:
`useTopicConsumerGroupGate.tsx`, lowercase `u`.

I'm on the fence about these two hooks, as they really are very similar. But as I said I think some duplication here is better than an abstraction.